compiler/src/dotty/tools/dotc/ast/Desugar.scala

```diff
@@ -915,20 +915,20 @@ object desugar {
                 report.error(i"right-associative extension method $problem", mdef.srcPos)
                 ext.paramss ++ mdef.paramss
               def noVParam = badRightAssoc("must start with a single parameter")
-              def checkVparam(params: ParamClause) = params match
```

Comment:
This needs a big comment with an example that explains what it does.

Reply:
This method no longer exists due to refactoring, but I have added more comments